dev-tools/typos.toml

```diff
@@ -1,10 +1,11 @@
 [default]
 extend-ignore-re = [
     "@author .*",
+    "(feature|minor) #[0-9]+ .+ (.*)",
+    "resolvement",
```

Comment:
i think it's used only once. maybe https://github.com/6b7562617765726c6f73/PHP-CS-Fixer/pull/3 ?

Reply:
```suggestion
```